doc/user/data/sql_funcs.yml

```diff
@@ -537,6 +537,9 @@
 
 - type: JSON
   functions:
+  - signature: jsonb_agg(expression) -> jsonb
+    description: Aggregate values (including nulls) as a jsonb array. ([docs](/sql/functions/jsonb_agg))
```

Comment:
```suggestion
  - signature: jsonb_agg(expression) -> jsonb
    description: Aggregate values (including nulls) as a jsonb array.
    url:  /docs/sql/functions/jsonb_agg
```

Reply:
See comment:

https://github.com/MaterializeInc/materialize/pull/20476#issuecomment-1631893912

Should I still go ahead and use the `url` attribute?

Reply:
Sounds like we need to fix it, like you suggested, rather than work around it. So — yes, please!